### PR TITLE
feat(inputs): SAP Horizon theme implementation

### PR DESCRIPTION
**Pull request by niyap**

### CODE REVIEW

**Comment by ndeshev** [comment]
> * Visual focus in the list: If you remember, I have told the team about that problem. Currently, we do not have a state to recognise that the visual focus is in the list.  It is not consistent with the Input component, because the implementation is not consistent. I use "focused" attribute to recognise when the active state should be applied. But for the Input component it is not removed, when you are navigating in the list while in the ComboBox component, it is removed. Both effect are not desired, the target design is to look like the input field is on hover, while you are navigating in the list, but that could not be achieved before we add a state for that. That is also discussed with Nikolay and a BLI will be created.

No need for BLI, once #3976 the focus property will behave as in ComboBox

**Comment by ndeshev** [comment]
There is a little bit of empty space between the suggestions popover and the focus outline. It occurs on every control, but it it is most noticeable in the nmore popup of the MultiInput, in it there is also the same problem on the top. 

image

**Comment by ndeshev** [comment]
There is 1px white space at the bottom of the value state messages, please ignore if it is from the Popup

image

File: packages/main/src/themes/sap_fiori_3_hcw/Token-parameters.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import "../base/YearPicker-parameters.css";
```
Comment (ndeshev) [inline packages/main/src/themes/sap_fiori_3_hcw/Token-parameters.css:1]: You should restore the YearPicker import

**Comment by ndeshev** [comment]
Looks good to me, with exception of the minor details regarding the implementation of the hover style to the inputs with focus in the popover, that we agreed to discuss, the change seems ready for merging.

File: packages/main/src/MultiComboBox.hbs [hunk 2]
```diff
@@ -54,6 +54,10 @@
 		aria-required="{{required}}"
 	/>
 
+	{{!-- {{#if this.readonly}}
+		<ui5-icon class="ui5-input-readonly-icon" name="edit" @click="{{_readonlyIconClick}}"></ui5-icon>
```
[the remark this answers is not in the document]
Comment (niyap) [reply packages/main/src/MultiComboBox.hbs:58]: I will change it, when the new icons are submitted. I left it in that way for testing purposes, in order to have a real icon if somebody wants to check how the icon will looks like.

**Review by elenastoyanovaa — APPROVED** [review]

File: packages/main/src/themes/sap_horizon/parameters-bundle.css [hunk 3]
```diff
@@ -27,7 +27,7 @@
 @import "../base/RadioButton-parameters.css";
 @import "./RatingIndicator-parameters.css";
 @import "./RadioButton-parameters.css";
-@import "../base/Select-parameters.css";
+@import "./Select-parameters.css";
```
[the remark this answers is not in the document]
Comment (niyap) [reply packages/main/src/themes/sap_horizon/parameters-bundle.css:30]: Yes, we need it. I have add a value to a variable in Select-parameters.css for SAP Horizon

**Review by ndeshev — APPROVED** [review]

**Pull request merged**